include/pmacc/PMaccConfig.cmake

```diff
@@ -141,6 +141,9 @@ if((NOT ALPAKA_ACC_GPU_CUDA_ENABLE) AND ALPAKA_ACC_GPU_CUDA_ONLY_MODE)
     message(WARNING "ALPAKA_ACC_GPU_CUDA_ONLY_MODE is set to OFF because cuda backend is not activated")
 endif()
 
+set(PMacc_LIBRARIES ${PMacc_LIBRARIES} ${cupla_LIBRARIES})
```

Comment:
I think so but I do not know the full context. When you do this you also have to remove all explicit setters for cupla_IncludeDir etc. If they exist.